minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -385,76 +391,12 @@ public static Container getContainer(String clusterId, String role) {
 
     public static String getContainerIp(String clusterId, String role) {
         Container container = getContainer(clusterId, role);
-        if ( container != null ) {
-            return DockerContainersUtil.getIpAddress( dockerClient, container.getId() );
+        if (container != null) {
+            return DockerContainersUtil.getIpAddress(dockerClient, container.getId());
         }
         return null;
     }
```

Comment:
Is getContainerIp still used? Callers should use AbstractContainer.getIpAddress()